third_party/googletest/src/googlemock/src/gmock-internal-utils.cc

```diff
@@ -158,13 +162,23 @@ GTEST_API_ void Log(LogSeverity severity, const std::string& message,
 
   if (severity == kWarning) {
     // Prints a GMOCK WARNING marker to make the warnings easily searchable.
+#if !GTEST_OS_STARBOARD
```

Comment:
These need a solution for Starboard. Shouldn't we have a `SbLogFormat` here in the #else clause ?